packages/builder/src/components/design/PropertiesPanel/PropertyControls/OptionsEditor/OptionsEditor.svelte

```diff
@@ -10,9 +10,15 @@
   let drawer
   let tempValue = value || []
 
-  const saveFilter = async () => {
-    // Filter out incomplete options
-    tempValue = tempValue.filter(option => option.value && option.label)
+  const saveOptions = async () => {
+    // Filter out incomplete options, default if needed
+    tempValue = tempValue.filter(option => option.value || option.label)
+    for (let i = 0; i < tempValue.length; i++) {
+      let option = tempValue[i]
+      option.label = option.label ? option.label : option.value
+      option.value = option.value ? option.value : option.label
```

Comment:
NAB: you don't need to update this, but just letting you know that you can use shorthand here to make this even terser
`option.label = option.label || option.value`

There is actually something called a "logical OR assignment" which is never used in the whole project so please don't introduce this, but you can make this *even shorter* by doing
`option.label ||= option.value`
which basically means "only assign `option.value` to `option.label` if `option.label` is falsey"

Alright that's my JS lesson over for the day :laughing:

Reply:
Nice! I forgot about that in JS. I had tried using '??' but that doesn't work for falsey values.

Reply:
Yep `??` only works for literal `null` or `undefined` whereas `||` works for anything falsey.